Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -128,7 +128,7 @@ private void ensurePowerModeForDrive() {
         ensureModeForTalon(rightDrive, TalonControlMode.PercentVbus);
     }
 
-    public void tankDriveVelocityPid(double leftPower, double rightPower) {
+    public void tankDriveVelocityPercentMax(double leftPower, double rightPower) {
```

Comment:
rename these arguments to something like `leftVelocityPercent` and `rightVelocityPercent`